src/brouwers/shop/templates/shop/index.html

```diff
@@ -0,0 +1,50 @@
+{% extends "shop/base.html" %}{% load i18n %}
+
+{% block content %}
+    <div class="row">
+
+        <div id="category-carousel" class="category__carousel carousel slide" data-ride="carousel">
+            <!-- Indicators -->
+            <ol class="carousel-indicators">
+                <li data-target="#category-carousel" data-slide-to="0" class="active"></li>
```

Comment:
these should be dynamically filled based on `carousel_images`, using `{{ forloop.counter0 }}`